docs/content/dagster-plus/deployment/azure/acr-user-code.mdx

```diff
@@ -89,18 +89,26 @@ First, we'll need to generate a service principal for GitHub Actions to use to a
 az ad sp create-for-rbac --name "github-actions-acr" --role contributor --scopes /subscriptions/<your_azure_subscription_id>/resourceGroups/<your_resource_group>/providers/Microsoft.ContainerRegistry/registries/<your_acr_name>
 ```
 
-This command will output a JSON object with the service principal details. Make sure to save the `appId`, `password`, and `tenant` values - we'll use them in the next step.
+This command will output a JSON object with the service principal details. Make sure to save the `appId` and `password` values - we'll use them in the next step.
```

Comment:
yea fair enough; but the tenant value does end up getting used later in the compute log manager docs

Reply:
did not actually use them.